<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>864</width>
    <height>665</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="NorthBtn">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EastBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WestBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SouthBtn">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="layoutImage">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mapLayout"/>
   </widget>
   <widget class="QPushButton" name="takeAllButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Take All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="take1Button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Take</string>
    </property>
   </widget>
   <widget class="QListWidget" name="inventoryList">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>180</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openCrafting">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Craft Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="placeButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Place</string>
    </property>
   </widget>
   <widget class="QListWidget" name="roomItemList">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>180</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
   <zorder>layoutImage</zorder>
   <zorder>NorthBtn</zorder>
   <zorder>EastBtn</zorder>
   <zorder>WestBtn</zorder>
   <zorder>SouthBtn</zorder>
   <zorder>label</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>takeAllButton</zorder>
   <zorder>take1Button</zorder>
   <zorder>inventoryList</zorder>
   <zorder>openCrafting</zorder>
   <zorder>placeButton</zorder>
   <zorder>roomItemList</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
